mmdet/core/bbox/assigners/uniform_assigner.py

```diff
@@ -0,0 +1,137 @@
+import warnings
+
+import torch
+
+from ..builder import BBOX_ASSIGNERS
+from ..iou_calculators import build_iou_calculator
+from ..transforms import bbox_xyxy_to_cxcywh
+from .assign_result import AssignResult
+from .base_assigner import BaseAssigner
+
+
+@BBOX_ASSIGNERS.register_module()
+class UniformAssigner(BaseAssigner):
+    """Uniform Matching between the anchors and gt boxes, which can achieve
+    balance in positive anchors.
+
+    Args:
+        pos_ignore_thresh (float): the threshold to ignore positive anchors
+        neg_ignore_thresh (float): the threshold to ignore negative anchors
+        match_times(int): Number of positive anchors for each gt box.
+           Default 4.
+        iou_calculator (dict): iou_calculator config
+    """
+
+    def __init__(self,
+                 pos_ignore_thresh,
+                 neg_ignore_thresh,
+                 match_times=4,
+                 iou_calculator=dict(type='BboxOverlaps2D')):
+        self.match_times = match_times
+        self.pos_ignore_thresh = pos_ignore_thresh
+        self.neg_ignore_thresh = neg_ignore_thresh
+        self.iou_calculator = build_iou_calculator(iou_calculator)
+
+    def assign(self,
+               bbox_pred,
+               anchor,
+               gt_bboxes,
+               gt_bboxes_ignore=None,
+               gt_labels=None):
+        warnings.warn('gt_bboxes_ignore was not considered for now.')
```

Comment:
gt_bboxes_ignore is always not considered. So this warning is unnecessary and can be put into the docstirng's Note section